compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -250,6 +294,71 @@ struct InsertQuantizeOp final : public luci::CircleNodeMutableVisitor<void>
   INSERT_QUANTIZE_TO_BINARY_OP(luci::CirclePow, x, y)
   INSERT_QUANTIZE_TO_BINARY_OP(luci::CircleSub, x, y)
 
+  // Multiple-output ops that receive one activation as inputs
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleSplit, input)
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleSplitV, input)
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleTopKV2, input)
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleUnique, input)
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleUnpack, value)
+
+  // Multiple-output ops that receive multiple activations as inputs
+  INSERT_QUANTIZE_MULTI_OUTPUT_NODES_WITH_INPUTS(luci::CircleIf, input, input_count)
+  INSERT_QUANTIZE_MULTI_OUTPUT_NODES_WITH_INPUTS(luci::CircleWhile, input, input_count)
+  INSERT_QUANTIZE_MULTI_OUTPUT_NODES_WITH_INPUTS(luci::CircleCustom, inputs, numInputs)
+
+  void visit(luci::CircleNonMaxSuppressionV4 *node)
```

Comment:
Also, please remove NonMaxSuppression Ops too. I'm not sure this is a correct way to insert quantize Ops. The ofm of NonMaxSuppression is indices, so inserting Quantize Op looks unnatural.

Let's support this when necessary.